src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AppealReviewOutcome.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.entities;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum AppealReviewOutcome {
```

Comment:
@srinivasuluputcha you will need to create an accompanying Test class.

Reply:
@bradsorour Added the coverage